assignments/lambda-classes.js

```diff
@@ -1 +1,114 @@
 // CODE here for your Lambda Classes
+
+// Create a Person Class
+class Person{
+    constructor(personAttributes){
+        this.name = personAttributes.name;
+        this.age = personAttributes.age;
+        this.location = personAttributes.location;
+    }
+//Method for Class Person
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+      }
+}
+
+//Create an Instructor Class
+class Instructor extends Person {
+    constructor(instructorAttributes) {
+        super(instructorAttributes);
+        this.specialty = instructorAttributes.specialty;
+        this.favLanguage = instructorAttributes.favLanguage;
+        this.catchPhrase = instructorAttributes.catchPhrase;
+    }
+//Method for Class Instructor
+    demo(subject) {
```

Comment:
Nice use of the parameter, but i want to see an use of `this` keyword to grab the attributes of our object